crates/autopilot/src/arguments.rs

```diff
@@ -414,15 +416,17 @@ impl FromStr for FeePolicyKind {
             "priceImprovement" => {
                 let factor = parts
                     .next()
-                    .ok_or("missing price improvement factor")?
+                    .context("missing price improvement factor")?
                     .parse::<f64>()
-                    .map_err(|e| format!("invalid price improvement factor: {}", e))?;
+                    .map_err(|e| anyhow::anyhow!("invalid price improvement factor: {}", e))?;
```

Comment:
You should be able to use `.context()` instead of `.map(anyhow::anyhow!())`.

Reply:
@MartinquaXD it is not possible to do that without losing the `e`, which we don't want to :thinking:

Reply:
`.context()` will wrap the error with the additional data you put into the string. Basically adding that to the error "call stack".
So I believe it should be fine but I also don't care too much about it. 👌